include/openthread/thread_ftd.h

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2016, The OpenThread Authors.
+ *  Copyright (c) 2017, The OpenThread Authors.
```

Comment:
Actually this is a new file, although github show it as modification. Could I confirm that should we use "2017" or "2016-2017" here?